security/cmd/istio_ca/main.go

```diff
@@ -183,6 +188,14 @@ func init() {
 	flags.DurationVar(&opts.probeCheckInterval, "probe-check-interval", defaultProbeCheckInterval,
 		"Interval of checking the liveness of the CA.")
 
+	// Generate webhook secrets
+	flags.BoolVar(&opts.genWebhookSecrets, "generate-webhook-secrets", false,
+		"Generate secrets for Istio webhook services.")
+	flags.StringVar(&opts.injectorWebhookService, "injector-webhook-service", "",
+		"Sidecar injector webhook service name.")
+	flags.StringVar(&opts.injectorWebhookSecret, "injector-webhook-secret", "",
+		"Sidecar injector webhook secret name.")
```

Comment:
I'd like to see a e2e design for how this is going to work? e.g. how does this scale to multiple webhooks? How do we install/webhook after the CA is already started?

Reply:
I think we only need to support two or three webhooks, so maybe we can just add them when needed. Or alternatively, we could have a list of strings as the flag. CA will store key, server cert and ca cert in the secret. The install tool can read from the secret for webhook registration, or the webhook service itself can do that if we implement the self-registration.